jmh/jmh.bzl

```diff
@@ -77,10 +77,12 @@ scala_generate_benchmark = rule(
 def scala_benchmark_jmh(**kw):
     name = kw["name"]
     deps = kw.get("deps", [])
+    runtime_deps = kw.get("runtime_deps", [])
```

Comment:
@donce I want to understand what are usecases when runtime deps are needed here. Do you have a real world example?

Reply:
We're benchmarking logging, which requires a custom logging config. The custom logging config is a static `xml` file with `resources` target, which is added to the benchmark.